File: go/1066.go
package main

import (
	"fmt"
	"math"
)

func main() {
  var n, par, impar, pos, neg int
  for i := 0; i < 5; i++ {
    fmt.Scanf("%d", &n)
    if n % 2 == 0 {
      par += 1
    }

    if math.Abs(float64(n % 2)) == 1 {
      impar += 1
    }

    if n > 0 {
      pos += 1
    }

    if n < 0 {
      neg += 1
    }
  }
  fmt.Printf("%d valor(es) par(es)\n%d valor(es) impar(es)\n%d valor(es) positivo(s)\n%d valor(es) negativo(s)\n", par, impar, pos, neg)
}

File: lua/1066.lua
local par, impar, pos, neg = 0, 0, 0, 0
local n

for i = 1, 5 do
    n = io.read("*n")

    if n % 2 == 0 then
        par = par + 1
    end

    if math.abs(n % 2) == 1 then
        impar = impar + 1
    end

    if n > 0 then
        pos = pos + 1
    end

    if n < 0 then
        neg = neg + 1
    end
end

print(string.format("%d valor(es) par(es)", par))
print(string.format("%d valor(es) impar(es)", impar))
print(string.format("%d valor(es) positivo(s)", pos))
print(string.format("%d valor(es) negativo(s)", neg))
